tests/test_mongodb_connection.py

```diff
@@ -0,0 +1,37 @@
+import unittest
+import hydra
+import mongomock
+
+from core_lib.data_transform.result_to_dict import result_to_dict
+from core_lib.helpers.config_instances import instantiate_config
+from core_lib.connection.mongodb_connection_registry import MongoDBConnectionRegistry
+hydra.core.global_hydra.GlobalHydra.instance().clear()
+hydra.initialize(config_path='./test_data/test_config')
+configfile = "test_mongodb.yaml"
+config = hydra.compose(configfile)
+
+
+class TestMongoDBConnection(unittest.TestCase):
+
+    @mongomock.patch(servers=((config.core_lib.mongodb.config.url.host, config.core_lib.mongodb.config.url.port), ))
+    def test_instantiate(self):
+        mongodb = instantiate_config(config.core_lib.mongodb)
+        self.assertIsInstance(mongodb, MongoDBConnectionRegistry)
+        with mongodb.get() as client:
+            collection = client.testing_collection.example

```

Comment:
who is insertion to the example collection? test is like guess work.  things should be explicit

Reply:
can you please elaborate a little more on this comment

Reply:
MongoDB will create the database and collection if it is not already present